ops/storage.py

```diff
@@ -299,11 +299,13 @@ def _save_notice_list(self, notices: '_Notices') -> None:
 
 
 # we load yaml.CSafeX if available, falling back to slower yaml.SafeX.
-_BaseDumper = getattr(yaml, 'CSafeDumper', yaml.SafeDumper)  # type: Type[yaml.SafeDumper]
-_BaseLoader = getattr(yaml, 'CSafeLoader', yaml.SafeLoader)  # type: Type[yaml.SafeLoader]
+_dtype = Union[Type[yaml.SafeDumper], Type[yaml.cyaml.CSafeDumper]]
+_ltype = Union[Type[yaml.SafeLoader], Type[yaml.cyaml.CSafeLoader]]
+_BaseDumper: _dtype = getattr(yaml, 'CSafeDumper', yaml.SafeDumper)
+_BaseLoader: _ltype = getattr(yaml, 'CSafeLoader', yaml.SafeLoader)
```

Comment:
I think these types don't actually help here (Pyright works without any types here).

If we do need them for some reason I'm missing, let's call them `_DumperType` and `_LoaderType`.